app/controllers/users_controller.rb

```diff
@@ -0,0 +1,48 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_user, except: [:show]
+  
+  def show
+    @history = OrderHistory.new(current_user)
+  end
+  
+  def edit
+  end
+
+  def update
+    if @user.update(user_params)
+      flash[:notice] = t('user.flashes.account_updated')
+      redirect_to edit_user_path
+    else
+      render :edit
+    end
+  end
+  
+  def update_password
+    if @user.update_with_password(user_params)
+      sign_in @user, bypass: true
+      flash[:notice] = t('user.flashes.password_updated')
+      redirect_to edit_user_path
+    else
+      render :edit
+    end
+  end
+  
+  private
+  
+  def find_user
+    @user = current_user    
```

Comment:
we can :) I removed unnecessary method.
P.S. I don't know why, but I thought that we can't use 'current_user' in view.